Automatically supply scenario path to Envision server (#1806)
huawei-noah/SMARTS: CHANGELOG.md

## [Unreleased]
### Added
### Changed

> - Scenario paths need not be supplied manually to Envision server while setup. Scenario paths are automatically sent to Envision server from SMARTS during simulation startup phase.

Reviewer: This is not quite the case.

```suggestion
- Scenario paths is no longer manually supplied to Envision server while setup. Scenario paths are automatically sent to Envision server from SMARTS during simulation startup phase.
```

Author: I have changed it.
